css_sass/README.md

```diff
@@ -0,0 +1,311 @@
+# Coveo CSS/Sass - Cloud Platform Standards {
+
+Other Standards
+
+- [TypeScript](../README.md)  
+- [React/Redux](../react_redux)
+
+## Table of Contents
+
+1. [General rules](#general-rules)
+1. [Terminology](#terminology)
+    - [Rule Declaration](#rule-declaration)
+    - [Selectors](#selectors)
+    - [Properties](#properties)
+1. [CSS](#css)
+    - [Formatting](#formatting)
+    - [Comments](#comments)
+    - [OOCSS and BEM](#oocss-and-bem)
+    - [ID Selectors](#id-selectors)
+    - [JavaScript hooks](#javascript-hooks)
+    - [Border](#border)
+1. [Sass](#sass)
+    - [Syntax](#syntax)
+    - [Ordering](#ordering-of-property-declarations)
+    - [Variables](#variables)
+    - [Mixins](#mixins)
+    - [Extend directive](#extend-directive)
+    - [Nested selectors](#nested-selectors)
+1. [Translation](#translation)
+
+## General Rules
+
+**Use [Vapor](https://github.com/coveo/vapor) style rules over anything else.**
+
+**Never place CSS styles in TypeScript files, even in React components.**
+
+**Never use inline style in HTML or EJS templates.**
+
+**Always place your style rules in a `.scss` file.**
+
+## Terminology
+
+### Rule declaration
+
+A “rule declaration” is the name given to a selector (or a group of selectors) with an accompanying group of properties. Here's an example:
+
+```css
+.listing {
+  font-size: 18px;
+  line-height: 1.2;
+}
+```
+
+### Selectors
+
+In a rule declaration, “selectors” are the bits that determine which elements in the DOM tree will be styled by the defined properties. Selectors can match HTML elements, as well as an element's class, ID, or any of its attributes. Here are some examples of selectors:
+
+```css
+.my-element-class {
+  /* ... */
+}
+
+[aria-hidden] {
+  /* ... */
+}
+```
+
+### Properties
+
+Finally, properties are what give the selected elements of a rule declaration their style. Properties are key-value pairs, and a rule declaration can contain one or more property declarations. Property declarations look like this:
+
+```css
+/* some selector */ {
+  background: #f1f1f1;
+  color: #333;
+}
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+## CSS
+
+### Formatting
+
+* Use soft tabs (2 spaces) for indentation
+* Prefer dashes over camelCasing in class names.
+* Avoid using ID selectors as much as possible
+* When using multiple selectors in a rule declaration, give each selector its own line.
+* Put a space before the opening brace `{` in rule declarations
+* In properties, put a space after, but not before, the `:` character.
+* Put closing braces `}` of rule declarations on a new line
+* Put blank lines between rule declarations
```

Comment:
I would also add:
* No trailing whitespaces.
* Never use tabs.
* Never have the opening bracket `{` alone on its line.